app/models/ems_refresh.rb

```diff
@@ -55,6 +55,10 @@ def self.queue_refresh(target, id = nil, sync = false)
     end
   end
 
+  def self.queue_event_for_refresh(event, ems)
+    queue_merge_async(event, ems, 'refresh_new')
```

Comment:
I don't think we want to merge these types of methods so you can just do an `MiqQueue.put` here